ward/cparser.lua

```diff
@@ -765,6 +765,7 @@ local grammar = pattern {
   float_type =
     keyword "float" * value(type_float) +
     keyword "double" * value(type_float) +
+    keyword "_Float128" * value(type_float) +
     keyword "long" * sp * rule "float_type",
```

Comment:
Strictly speaking, this rule is now no longer right -- at least I assume there is no `long _Float128` ? But I guess we don't need to worry, as at worst it'll accept some code that the compiler than later rejects.

(Though, fun fact: In C, `long double` and `double long` are both valid, so strictly speaking, that rule always was too narrow... but hey, who cares. After all, also `double const long` and `const long double` are equivalent, and I am not sure ward is prepared to deal with *that*... but again, I don't mind, just pointing it out ;-)